spacy/tests/regression/test_issue5152.py

```diff
@@ -1,3 +1,6 @@
+# coding: utf8
+from __future__ import unicode_literals
+
```

Comment:
@ines : having dropped Python 2, we don't need these coding annotations and future imports anymore, right? Checking across the `develop` branch, it looks like these have crept back in from the merges from `master`...

Reply:
Good catch, thanks! And yeah, I think if it's new files, the merge will just port them over as-is and then we have to go hunt for them ourselves. But those should definitely all go!

Reply:
I can take care of that 🙂